package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chitragupta",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
Why changing version to 1.1.0 and not 1.0.3?

Reply:
This is an addition of a new feature